src/tag/Type.h

```diff
@@ -46,6 +46,7 @@ enum TagType
 	TAG_NAME,
 	TAG_GENRE,
 	TAG_DATE,
+	TAG_ORIGINAL_DATE,
```

Comment:
I wasn't able to get a preprocessor check here without getting an error. If this line of code needs a check, please help me with it.